cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/service/http/HttpHandlerGenerator.java

```diff
@@ -542,6 +546,14 @@ private void generateTransactionalDelegateBody(GeneratorAdapter mg, Method metho
                        Methods.getMethod(TransactionContext.class, "getTransactionContext"));
       mg.storeLocal(txContext, txContextType);
 
+      // HttpServiceResponder wrappedResponder = wrapResponder(responder);
+      int wrappedResponder = mg.newLocal(serviceResponderType);
+      mg.loadThis();
```

Comment:
mg = Method Generator. It's for generating the method (in ASM example is called mg, hence we just used it. I am open to call it `methodGenerator` if that's clearer).